### PR TITLE
Online mode

### LINKED ISSUE #45: Online mode
Currently the explorer executable expects data in the store to be updated by another process (that would be usually a game app).  In order to be easier to run an explorer instance standalone, add options like `--online`/`--seed=…` to run a `Swarm` instance in the background and update date by itself.

### PR DESCRIPTION
**Pull request by dahlia**
This patch adds several CLI options like `-s`/`--seed` and `-I`/`--ice-server`.   Please see also commit messages.

Closes #45.

### CODE REVIEW

File: Libplanet.Explorer.Executable/Program.cs [hunk 1]
```diff
@@ -18,12 +28,97 @@ public static void Main(string[] args)
...
+
+            Task swarmTask = Task.Run(
+                async () =>
+                {
+                    if (swarm is null)
+                    {
+                        return;
+                    }
+
+                    HashSet<Peer> peers = new HashSet<Peer>();
```
Comment (longfin) [inline Libplanet.Explorer.Executable/Program.cs]:
```suggestion
                    var peers = new HashSet<Peer>();
```

File: Libplanet.Explorer.Executable/Program.cs [hunk 2]
```diff
@@ -18,12 +28,97 @@ public static void Main(string[] args)
...
+                cts.Token
+            );
+
+            try
+            {
+                Task.WaitAll(webHost.RunAsync(cts.Token), swarmTask);
+            }
+            catch (OperationCanceledException)
+            {
+                if (swarm != null)
```
Comment (longfin) [inline Libplanet.Explorer.Executable/Program.cs]:
```suggestion
                if (!(swarm is null))
```

or 

```suggestion
                if (swarm is Swarm<AppAgnosticAction>)
```

File: Libplanet.Explorer/Interfaces/IBlockChainContext.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+        public static Schema GetSchema<T>(this IBlockChainContext<T> context)
+            where T : IAction, new()
+        {
+            return _schemaObjects.GetValue(
+                context,
+                (_) =>
+                {
+                    var s = new Schema { Query = new BlocksQuery<T>(context.BlockChain) };
+                    System.Console.WriteLine("\n\t{0}\n", s);
```
Comment (longfin) [inline Libplanet.Explorer/Interfaces/IBlockChainContext.cs]: Is it intended?

File: Libplanet.Explorer.Executable/Options.cs [hunk 4]
```diff
@@ -1,17 +1,127 @@
...
+                Seed = new Peer(pubkey, endpoint);
+            }
+        }
+
+        public Peer Seed { get; set; }
+
+        [Option(
+            'I',
+            "ice-server",
+            Required = true,
```
Comment (earlbread) [inline Libplanet.Explorer.Executable/Options.cs]: Isn't this required only when a seed node exists?

Comment (dahlia) [reply Libplanet.Explorer.Executable/Options.cs]: You're right.  I'm going to amend.

File: Libplanet.Explorer/Interfaces/IBlockChainContext.cs (hunk 3 above)
Comment (dahlia) [reply Libplanet.Explorer/Interfaces/IBlockChainContext.cs]: No, I'm going to get rid of this line.

**Review by earlbread — APPROVED** [review]

**Review by longfin — APPROVED** [review]

**Review by moreal — APPROVED** [review]

**Pull request merged**